tests/ecmp/inner_hashing/test_wr_inner_hashing.py

```diff
@@ -22,6 +22,7 @@ class TestWRDynamicInnerHashing():
 
     @pytest.fixture(scope="class", autouse=True)
     def setup_dynamic_pbh(self, request):
+        request.getfixturevalue("save_config_teardown")
```

Comment:
I don't get why we need to do a config save at all, since warm reboot gets the current running config and does not really load config from persisted config we shouldn't need to do a config save at all as far as I see.

Reply:
removed the saving of configurations before WR and save configurations as a teardown